static/src/stylesheets/module/email-signup/_newDesign.scss

```diff
@@ -0,0 +1,148 @@
+/* Newsletter Description */
+
+$base-margin: 8px;
+$desktop-img-size: 200px;
+$mobile-img-size: 100px;
+
+.newsletter-embed {
+    * {
+        box-sizing: border-box;
+    }
+    height: 100%;
+    overflow: hidden;
+}
+.newsletter-embed__image {
+    max-width: $mobile-img-size;
+    float: left;
+    margin: $base-margin;
+}
+
+.newsletter-embed__image img {
+    width: 100%;
+}
+
+.newsletter-embed__body {
+    margin: $base-margin;
+}
+
+.newsletter-embed__summary {
+    font-family: 'Guardian Egyptian Web', Georgia, serif;
+}
+
+.newsletter-embed__headline-wrapper {
+    margin: 0 0 $base-margin;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    height: $mobile-img-size;
+}
+
+.newsletter-embed__image + .newsletter-embed__headline-wrapper {
+    padding-left: $base-margin;
+}
+
+.newsletter-embed__headline {
+    font-size: 24px;
+    font-weight: 700;
+    line-height: 1;
+}
+
+.newsletter-embed__description {
+    /* Small hack to get description to go under image if close to fitting under */
+    padding-top: 2px;
+    font-size: 16px;
+    clear: left;
+}
+
+/* Sign up Form */
+.newsletter-embed__form {
+    clear: left;
+    font-size: 16px;
+    font-family: 'Guardian Text Sans Web', SansSerif;
+}
+
+.newsletter-embed__form-wrapper {
+    max-width: 500px;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: auto;
+    grid-template-areas:
```

Comment:
As a relative CSS beginner this blows my mind 🤯

Reply:
https://css-tricks.com/snippets/css/complete-guide-grid/ was my best friend for this